test/unit/modules/util/next-tick.spec.js

```diff
@@ -0,0 +1,25 @@
+import { nextTick } from 'core/util/env'
+
+describe('nextTick', () => {
+  it('accepts a callback', done => {
+    nextTick(done)
```

Comment:
I'm not sure I understand what you're saying, but are you suggesting having a separate `nextTickPromise` method or similar?